src/packages/user/user/collection/views/grid/user-grid-collection-view.element.ts

```diff
@@ -2,37 +2,51 @@ import { getDisplayStateFromUserStatus } from '../../../../utils.js';
 import { UmbUserCollectionContext } from '../../user-collection.context.js';
 import { type UmbUserDetail } from '../../../types.js';
 import { css, html, nothing, customElement, state, repeat, ifDefined } from '@umbraco-cms/backoffice/external/lit';
-import { UmbTextStyles } from "@umbraco-cms/backoffice/style";
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
 import { UMB_COLLECTION_CONTEXT } from '@umbraco-cms/backoffice/collection';
 import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
-import { UserStateModel } from '@umbraco-cms/backoffice/backend-api';
+import { UserGroupResponseModel, UserStateModel } from '@umbraco-cms/backoffice/backend-api';
+import { UmbUserGroupCollectionRepository } from '@umbraco-cms/backoffice/user-group';
 
-@customElement('umb-user-collection-grid-view')
-export class UmbUserCollectionGridViewElement extends UmbLitElement {
+@customElement('umb-user-grid-collection-view')
+export class UmbUserGridCollectionViewElement extends UmbLitElement {
 	@state()
 	private _users: Array<UmbUserDetail> = [];
 
 	@state()
 	private _selection: Array<string> = [];
 
+	@state()
+	private _loading = false;
+
+	#userGroups: Array<UserGroupResponseModel> = [];
 	#collectionContext?: UmbUserCollectionContext;
+	#userGroupCollectionRepository = new UmbUserGroupCollectionRepository(this);
 
 	constructor() {
 		super();
 
-		//TODO: Get user group names
-
 		this.consumeContext(UMB_COLLECTION_CONTEXT, (instance) => {
 			this.#collectionContext = instance as UmbUserCollectionContext;
 			this.observe(this.#collectionContext.selection, (selection) => (this._selection = selection));
```

Comment:
note about missing ControllerAlias, to make observe replace it self if this method is called again.